www/docs/guide/standalone-deployment.md

```diff
@@ -21,17 +21,83 @@ A list of available base images can be found in
 `hass-addon-sunsynk-multi/build.yaml` and `hass-addon-mbusd/build.yaml`.
 Use the one that is most appropriate for your host computer.
 
-## Sunsynk Multi
+### Sunsynk Multi
 
 * Copy `options.yaml.template` to `options.yaml` and make changes to `options.yaml` to match your setup. Use `PORT: tcp://mbusd:502` for the inverter port if you want to use the `mbusd` included in this docker compose stack.
 * Build the image `BUILD_FROM=<base_image> docker compose build sunsynk-multi`
 * Run the container `docker compose up -d sunsynk-multi`
 * See the container logs `docker compose logs -f sunsynk-multi`
 
-## Mbusd
+### Mbusd
 
 * Edit `docker-compose.yaml` changing the values under `environment` to match your configuration, leaving the device set to `/dev/ttyUSB0` as we mount the correct port to this location in the next step.
 * Under `volumes` change `/dev/ttyRS485` to the RS485 port of your host computer.
 * Build the image `BUILD_FROM=<base_image> docker compose build mbusd`
 * Run the container `docker compose up mbusd`
 * View container logs `docker compose logs -f mbusd`
+
+## Using Pre-built Docker Images
+The repo also contains prebuilt Docker images for Sunsynk Multi. You can see the different images for the various supported architectures [here](https://github.com/kellerza?tab=packages&repo_name=sunsynk).
+
+### Docker-Compose examples:
+#### amd64
+``` yaml
+services:
+  sunsynk-multi:
+    restart: unless-stopped
+    image: ghcr.io/kellerza/hass-addon-sunsynk-multi/amd64:stable
+    volumes:
+      - ${PWD}/options.yaml:/usr/src/.local.yaml
```

Comment:
Should this not bind to `/data/options.yaml`?

```suggestion
      - ${PWD}/options.yaml:/data/options.yaml
```

Reply:
Yes! You are correct. will update.